database/postgresSetup.sql

```diff
@@ -32,7 +32,7 @@ CREATE TABLE IF NOT EXISTS events(
       user_id int REFERENCES users,
       PRIMARY KEY (feed_id, user_id));
 
-INSERT INTO users(username, password) values ('foo', 'bar');
+INSERT INTO users(username, password) values ('foo', crypt('bar', gen_salt('md5')));
```

Comment:
@jtfennell suggested that we start hashing passwords, which is a great idea. This script populates the DB with sample data.